cli/src/commands/mod.rs

```diff
@@ -485,6 +485,9 @@ struct DuplicateArgs {
 /// Abandon a revision, rebasing descendants onto its parent(s). The behavior is
 /// similar to `jj restore --changes-in`; the difference is that `jj abandon`
 /// gives you a new change, while `jj restore` updates the existing change.
+///
+/// Note that if you abandon the working-copy commit, a new empty working-copy commit will be created
+/// If you want to remove empty commit from the top of history, use jj unstash.
```

Comment:
You meant `unsquash`, not `unstash`. More importantly, I don't see when users would want to do that. It seems to me like it would usually be a mistake (you might end up accidentally modifying a commit you shouldn't be modifying). I'd rather we just delete the sentence.

Maybe the tutorial needs to emphasize that empty working-copy commits are normal and don't need to be "cleaned up"?

Reply:
Ok, I might change this to `edit @-` as well.
The point is not cleaning up, it's continuing working on a change as a working-copy commit.

Reply:
I'm not sure that workflow is common enough to mention it here (it seems to me that it's likely more common after squashing), but fair enough, we can always revert this change later if we hear that it confuses people.